src/sentry/search/events/fields.py

```diff
@@ -75,12 +77,13 @@ def __init__(self, name, alias, expression=None, expression_fn=None, result_type
 
         self.validate()
 
-    def get_expression(self, params) -> list[Any] | tuple[Any]:
-        if isinstance(self.expression, (list, tuple)):
+    def get_expression(self, params) -> list[Any] | None:
+        if self.expression is not None:
```

Comment:
this is technically a tiny behaviour change -- but from all the callers I can find they never pass in tuple and only list or None for expression